### PR TITLE
[INTERNAL] TaskRunner: Use new 'perf' log level instead of env variable

### PR DESCRIPTION
**Pull request by RandomByte**

### CODE REVIEW

**Comment by flovogt** [comment]
Do we not have to wait for the next logger release and then increase the logger dependency here before merging?

**Comment by RandomByte** [comment]
Already released: https://github.com/SAP/ui5-logger/releases/tag/v3.0.1-alpha.2

**Review by flovogt — APPROVED** [review]
LGTM

**Pull request merged**
